src/Validator/HttpUserAgent.php

```diff
@@ -36,17 +20,7 @@ public function __construct($data = null)
      */
     public function isValid(): bool
     {
-        $userAgent = $_SERVER['HTTP_USER_AGENT'] ?? null;
-
-        return $userAgent === $this->getData();
-    }
-
-    /**
-     * Retrieve token for validating call
-     */
-    public function getData(): mixed
-    {
-        return $this->data;
+        return $this->initial->userAgent === $this->current->userAgent;
```

Comment:
Not knowing this library well, is a situtation where `null === null` acceptable here?

It stands to reason that if a user has enabled this validator, then a `null` user-agent string would be a validation issue… maybe?

Reply:
OK, I just noticed the test case that asserts a `null` user-agent is deemed valid.

As you probably know this library better, is that behaviour still correct?